src/Model/Post/Engagement.php

```diff
@@ -167,61 +184,61 @@ private static function getSearchTextForItem(array $item): string
 
 	private static function getSearchText(array $item, array $receivers, array $tags): string
 	{
-		$body = '[nosmile]network:' . $item['network'];
+		$body = '[nosmile]network_' . $item['network'];
```

Comment:
Why are these search criterions escaped in advance? Isn't it the job of `Post\Engagement->escapeKeywords()`?

Reply:
We have to work with `_` instead of `:` since the full text search in MariaDB treats the `:` as word delimiter, but not the `_`.

Reply:
Wasn’t it the case before?

Reply:
Yeah. When I tested the index, I discovered that search results not always had been correct. Do I dug deeper and discovered a problem with `:` as a separator.